src/strongbox/core.clj

```diff
@@ -589,21 +612,31 @@
 ;;
 
 (defn-spec get-catalogue-location (s/or :ok :catalogue/location, :not-found nil?)
+  "returns the 'catalogue-location' map for the given `catalogue-name`.
+  returns the catalogue-location map for the currently selected catalogue by default.
+  returns `nil` if catalogue not found or no catalogue selected and no `catalogue-name` given."
   ([]
    (get-catalogue-location (get-state :cfg :selected-catalogue)))
   ([catalogue-name keyword?]
-   (->> (get-state :cfg :catalogue-location-list) (filter #(= catalogue-name (:name %))) first)))
+   (->> (get-state :cfg :catalogue-location-list)
+        (filter #(-> % :name (= catalogue-name)))
+        first)))
+
+(defn-spec default-catalogue (s/or :ok :catalogue/location, :not-found nil?)
```

Comment:
tests

Reply:
actually, I think this gets plenty of coverage from existing tests

Reply:
er, nope:

![Screenshot at 2021-09-26 13-22-13](https://user-images.githubusercontent.com/48790775/134792932-1b7d858c-4ba7-4eb6-9676-2f6d88d8f9f9.png)